Header header

std_msgs/UInt8 signal_battery
std_msgs/UInt8 power_battery
float32 current_right
float32 current_left
float32 current_sum
float32 current_joint0
float32 current_joint1
float32 current_joint2
int16 torque_right
int16 torque_left
int16 torque_joint0
int16 torque_joint1
int16 torque_joint2
